draft-js-mention-plugin/src/defaultRegExp.js

```diff
@@ -1,8 +1,30 @@
-// common chinese symbols: \u4e00-\u9eff - http://stackoverflow.com/a/1366113/837709
-// hiragana (japanese): \u3040-\u309F - https://gist.github.com/ryanmcgrath/982242#file-japaneseregex-js
-// katakana (japanese): \u30A0-\u30FF - https://gist.github.com/ryanmcgrath/982242#file-japaneseregex-js
-// For an advanced explaination about Hangul see https://github.com/draft-js-plugins/draft-js-plugins/pull/480#issuecomment-254055437
-// Hangul Syllables (korean): \uAC00-\uD7A3 - https://en.wikipedia.org/wiki/Korean_language_and_computers#Hangul_in_Unicode
-// Hangul Jamo (korean): \u3130-\u318F - https://en.wikipedia.org/wiki/Korean_language_and_computers#Hangul_in_Unicode
-// Cyrillic symbols: \u0410-\u044F - https://en.wikipedia.org/wiki/Cyrillic_script_in_Unicode
-export default '[\\w\u4e00-\u9eff\u3040-\u309F\u30A0-\u30FF\uAC00-\uD7A3\u3130-\u318F\u0410-\u044F]*';
+export default '[' +
+  'a-zA-Z0-9-' +
```

Comment:
Any specific reason why to support `-` instead of `_`? Why not support both?

Reply:
My reasoning was that `-` is used in e.g. Spanish names, but `_` is not used in real names. 

No harm in supporting both though and enables usernames with underscore. I'll add the changes.

Reply:
great thanks … makes sense for names. Nowadays this plugin is use by people for way more. I think supporting both out of the box makes sense to avoid confusion